tests/test_methods_and_attributes.py

```diff
@@ -111,6 +111,11 @@ def test_property_return_value_policies(access):
     assert getattr(obj, access + "_func").value == 1
 
 
+@pytest.mark.parametrize("access", ["static_ro", "static_rw"])
+def test_property_return_value_policies_static(access):
+    test_property_return_value_policies(access)
```

Comment:
I guess this can be merged back with the test above now that PyPy doesn't need to skip the static part.